crates/forge-runner/src/sierra_casm_runner.rs

```diff
@@ -0,0 +1,550 @@
+use blockifier::execution::syscalls::hint_processor::SyscallHintProcessor;
```

Comment:
I mentioned this to @Arcticae already, we should consider outsourcing it to the compiler a high priority. It is extremaly problematic to debug and maintain if something changes (speaking from experience)